server/.gitignore

```diff
@@ -0,0 +1,6 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

Comment:
`.gitignore` file allows us to state which files / folders should be ignored by git (notice that `/node_modules` folder isn't committed here.